pkg/network/ebpf/c/tracer-stats.h

```diff
@@ -0,0 +1,105 @@
+#ifndef __TRACER_STATS_H
```

Comment:
Moving common map access to a shared H file is a good refactor, but makes this harder to follow. In the future please do this in a separate PR.